src/explore-education-statistics-admin/src/pages/users/UserInvitePage.tsx

```diff
@@ -91,67 +120,175 @@ const UserInvitePage = ({
-        >
-          {() => {
-            return (
-              <Form id={formId}>
-                <FormFieldset
-                  id="email-fieldset"
-                  legend="Provide the email address for the user"
-                  legendSize="m"
-                  hint="The invited user must have a @education.gov.uk email address"
-                >
-                  <FormFieldTextInput<FormValues>
-                    label="User email"
-                    name="userEmail"
-                  />
-                </FormFieldset>
-
-                <FormFieldset
-                  id="role-fieldset"
-                  legend="Role"
-                  legendSize="m"
-                  hint="The users' role within the service."
-                >
-                  <FormFieldSelect<FormValues>
-                    label="Role"
-                    name="selectedRoleId"
-                    placeholder="Choose role"
-                    options={model?.roles.map(role => ({
-                      label: role.name,
-                      value: role.id,
-                    }))}
-                  />
-                </FormFieldset>
-
-                <Button type="submit" className="govuk-!-margin-top-6">
-                  Send invite
-                </Button>
-                <div className="govuk-!-margin-top-6">
-                  <ButtonText onClick={cancelHandler}>Cancel</ButtonText>
-                </div>
-              </Form>
-            );
-          }}
-        </Formik>
-      )}
-    </Page>
+        </div>
+        {!isLoading && (
+          <Formik<FormValues>
+            enableReinitialize
+            initialValues={{
+              userEmail: '',
+              selectedRoleId:
+                orderBy(model?.roles, role => role.name)?.[0]?.id ?? '',
+              selectedUserReleaseRoles: [],
+
+              selectedReleaseId: '',
+              selectedReleaseRole: '',
+            }}
+            validationSchema={Yup.object<FormValues>({
+              userEmail: Yup.string()
+                .required('Provide the users email')
+                .email('Provide a valid email address'),
+              selectedRoleId: Yup.string().required('Choose role for the user'),
+              selectedUserReleaseRoles: Yup.array(),
+
+              selectedReleaseId: Yup.string(),
+              selectedReleaseRole: Yup.string(),
+            })}
+            onSubmit={handleSubmit}
+          >
+            {form => {
+              return (
+                <Form id={formId}>
+                  <FormFieldset
+                    id="email-fieldset"
+                    legend="Provide the email address for the user"
+                    legendSize="m"
+                    hint="The invited user must have a @education.gov.uk email address"
+                  >
+                    <FormFieldTextInput<FormValues>
+                      label="User email"
+                      name="userEmail"
```

Comment:
This field is way too wide for the content expected in it. Setting `width={20}` would be more appropriate.